docs/charter.md

```diff
@@ -10,7 +14,43 @@ ICU4X will be built from the start with several key design constraints:
 3. Availability and ease of use in multiple programming languages.
 4. Written by i18n experts to encourage best practices.
 
-ICU4X will provide an API surface in the target client-side platforms, including the web platform, iOS, Android, WearOS, WatchOS, Flutter, and Fuchsia, supported in programming languages including Rust, JavaScript, Objective-C, Java, Dart, and C++.
+# Scope
+
+ICU4X aims to provide a set of APIs sufficient to internationalize basic modern software.
+
+With the focus on client-side, it will initially focus on the scope covered by the ECMA-402 extended to cover needs of more low level systems such as Gecko or Fuchsia.
+
+The current target is to cover all of ECMA-402 needs, and a modern subset of ICU4X APIs needed for software platforms such as Fuschia and Gecko.
+
+One of the benchmarks on decisions related to whether a given API belongs to ICU4X will be if it requires Unicode/CLDR data and would benefit from being managed together with other APIs that rely on the same version of Unicode and CLDR data.
+
+For ICU4C APIs and other internationalization needs that will not end up being included, ICU4X aims to provide a sufficient access to building blocks to allow third-party libraries to be developed that can interoperate with ICU4X with minimum data and logic duplication.
+
+In some rare cases, ICU4X may diverge from both ECMA-402, and ICU4C. Such diversions should be the result of improved API design and should allow users to cover the same needs with the upgraded APIs.
```

Comment:
first paragraph duplicates previous one

```suggestion
Two of the benchmarks for inclusion in ICU4X are: (1) if the API requires Unicode/CLDR data and (2) if it would benefit from being managed with other APIs that rely on the same version of Unicode and CLDR data.

For such APIs and functionality that will not end up being included, ICU4X aims to provide the building blocks to enable third-party libraries to be developed which depend on and interoperate with ICU4X, with minimum data and logic duplication.

In some rare cases, ICU4X may diverge from both ECMA-402, and ICU4C/ICU4J. Such diversions should be the result of improved API design and should allow users to cover the same needs with the upgraded APIs.
```